languages/python/reference/exercise-concepts/robot-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Concepts for robot-simulator
+
+[Example Implementation](https://github.com/exercism/python/blob/master/exercises/robot-simulator/example.py)
+
+
+## General
+
+- contstants: Are not enforced by the runtime, but are used via the convention of `<ALL_CAPS_NAMING>` to signal that these values are expected to remaine unchanged.  Perferably, constants are defined at a module level. The example solution uses the ALL_CAPS convention to define NORTH, SOUTH, EAST, and WEST constants. 
```

Comment:
Spelling: "contstants" should be "constants",  "remaine" should be "remain", "Perferably" should be "preferably". Also personally I'd think it would be good for us to use a conventional naming (ie **UPPER_CASE**, **snake_case**, **CamelCase**, **dromedaryCase** and **kebab-case**), rather than <ALL_CAPS_NAMING> the first time and ALL_CAPS the second.

Reply:
GAH.  I truly apologies for the "creative" spelling throughout.  Stupid of me to think I could do this directly in the GitHub interface and not get into trouble.  Spelling and other edits in the updated file.  Will "resolve" each as updates are made.